packages/editor/src/components/visual-editor/index.js

```diff
@@ -462,7 +462,13 @@ function VisualEditor( {
 								renderingMode !== 'post-only' ||
 									isDesignPostType
 									? 'wp-site-blocks'
-									: `${ blockListLayoutClass } wp-block-post-content` // Ensure root level blocks receive default/flow blockGap styling rules.
+									: `${ blockListLayoutClass } wp-block-post-content`, // Ensure root level blocks receive default/flow blockGap styling rules.
+								{
+									'has-global-padding':
```

Comment:
Just to confirm, this is to make things prettier when writing posts (without template preview?)

Things are working as expected for me. 

I can see that `has-global-padding` is on the root container in the editor, even if I switched "Inner blocks use content width" off for the content block:

| Before | After |
|--------|--------|
| <img width="200" alt="Screenshot 2024-10-31 at 4 08 15 pm" src="https://github.com/user-attachments/assets/206362f5-cc40-4f91-8a00-01198158b7fa"> | <img width="200" alt="Screenshot 2024-10-31 at 4 09 17 pm" src="https://github.com/user-attachments/assets/e88227d2-3717-46a1-bd94-b2a9957861cd"> | 


I tested this also on TT4 and could see the same thing.

Reply:
Thanks for taking this for a spin!

> Just to confirm, this is to make things prettier when writing posts (without template preview?)

Yep, that's the goal! Feedback from @jasmussen has been that we should never (or possibly almost never) have edge-to-edge content in the abstracted editor (without template preview), so this PR helps address a particular case where there was missing padding.

Ever since the editors were unified and folks can toggle template preview, there's a little less pressure for the editor to try to be WYSIWYG, as that's what template preview is for. So, we can afford to be a little less WYSIWYG to optimise for the writing experience / things looking neat.

But let's see what the designers think 🙂

Reply:
Great thanks, LGTM in that case